src/Strava/API/Service/REST.php

```diff
@@ -92,14 +105,24 @@ public function getAthlete($id = null)
             $path = 'athletes/' . $id;
         }
         $parameters['query'] = ['access_token' => $this->getToken()];
-        return $this->getResponse('GET', $path, $parameters);
+
+        $response = $this->getResponse('GET', $path, $parameters);
+
+        if ($this->responseVerbosity == 0)
+            return $response["body"];
```

Comment:
Shouldn't the return value and the if statement be the other way around if we want to maintain the original functionality for non-api breaking changes?
Originally the methods return the whole $response object, not just $response['body']

Reply:
Perhaps changing the comparison operator everywhere to !== is sufficient.

Reply:
I believe the original code only returned the body

Reply:
I've double-checked, the current version returns `json_decode($response->getBody(), JSON_PRETTY_PRINT)`, which in my change is now contained in `$expandedResponse['body']`. I believe this is correct.

Reply:
thanks, I'll do some testing later and try to merge it asap

Reply:
Hey @vredeling, did you get around to testing this?

Reply:
Just merged it... after that I move the multiple occurrences of `if ($this->responseVerbosity === 0) return $response["body"];` into the getResponse method.

Reply:
See 791946664ca603ac279c2430417e9878561e6fe0

Reply:
> Just merged it... after that I move the multiple occurrences of `if ($this->responseVerbosity === 0) return $response["body"];` into the getResponse method.

Yes, that's much nicer! Thanks 👍